119 withdrawal table split into withdrawalqueued withdrawalcompleted

gowthamsundaresan opened the pull request.

gowthamsundaresan commented:
Do `startBlock` and `createdAtBlock` in the `WithdrawalsQueued` table serve the same purpose? If so, we can get remove one of them

uditdc commented:
Thanks! 
Just one more thing, with the recent BigInt global JSON conversion, we no longer need to encapsulate createdAtBlock and updatedAtBlock under the "Number()" constructor.

gowthamsundaresan commented:
Sure!
- removed Number() constructors
- removed `startBlock` from schema and consequently from `seedWithdrawalsQueued()` and replaced its usage with `createdAtBlock` in withdrawal & staker routes
